webplatformcompat/models.py

```diff
@@ -349,7 +416,8 @@ def post_save_update_cache(sender, instance, created, raw, **kwargs):
         delay_cache = getattr(instance, '_delay_cache', False)
         if not delay_cache:
             from .tasks import update_cache_for_instance
-            update_cache_for_instance(name, instance.pk, instance, False)
+            update_cache_for_instance(
+                name, instance.pk, instance, update_only=False)
```

Comment:
Seems to call `update_cache_for_instance` with `update_only=False` a lot ... should we change the default parameter value to `False`?